### PR TITLE
Updated mpb bar progress bars

### PR DESCRIPTION
**Pull request by joereuss12**
This fixes a bug when downloading from an origin for the first time and a weird error message shows up. Turns out the package needed some updating. This took some time since the function we originally had for v7 no longer existed in v8. Also changed around the styling a tad and made it look nice and work with this new version upgrade. Removed what I thought was an unnecessary extra bar being made to say "done". Also found a typo to be fixed within handle_http.

### CODE REVIEW

**Comment by jhiemstrawisc** [comment]
Hey @joereuss12, it looks like something that was changed is causing the Test action to throw this:
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x8115ec]
```

Can you investigate and let me know when you've patched things back up?

**Comment by joereuss12** [comment]
@jhiemstrawisc just updated with some more changes too. I can undo the log level if needed depending on how Brian responds to the issue I made (#135) and I'll make a note in the issue to revert it back if needed.

File: handle_http.go [hunk 1]
```diff
@@ -596,30 +592,20 @@ Loop:
...
-						mpb.AppendDecorators(
-							decor.OnComplete(decor.Percentage(decor.WC{W: 5}), ""),
-						),
-					)
-					progressBar.SetTotal(resp.Size, true)
-					cancelledProgressBar.SetTotal(resp.Size, true)
+					progressBar.Abort(true)
+					log.Errorln("cancelled, too slow!")
+					log.Info("retrying...")
+					//cancelledProgressBar.SetTotal(resp.Size, true)
```
Comment (bbockelm) [inline handle_http.go:608]: Isn't there a higher function that emits an error message?  Seems strange to do that from the progress bar.

**Review by bbockelm — CHANGES_REQUESTED** [review]
The big item to focus on is how the `EwmaIncrement` method should be invoked.  It certainly looks wrong right now, always being called with 0.

The other items are mostly minor.

How do we test this functionality?

File: handle_http.go [hunk 2]
```diff
@@ -596,30 +592,20 @@ Loop:
 				if resp.Duration() < time.Second*time.Duration(slowTransferRampupTime) {
 					continue
 				} else if startBelowLimit == 0 {
-					log.Warnln("Download speed of ", resp.BytesPerSecond(), "bytes/s", " is below the limit of", downloadLimit, "bytes/s")
+					//log.Warnln("Download speed of ", resp.BytesPerSecond(), "bytes/s", " is below the limit of", downloadLimit, "bytes/s")
```
[the remark this answers is not in the document]
Comment (joereuss12) [reply handle_http.go:595]: I'm not super sure how that ended up commented out. Fixing now, sorry!

**Pull request merged**